### PR TITLE
 Added About Screen

### LINKED ISSUE #162: Add about screen
Add about screen in the app to display important information like:

* Developer name/profile
* Version
* Libraries used
* Report issue
* Contact
* GitHub repository link
* Google Play/F-Droid link
* Contributor list
* License/Restrictions
Etc.

### PR DESCRIPTION
**Pull request by shiv07tiwari**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
If there are any UI change, please include the screenshots also.

Fixes #162

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
Screenshot.

**Comment by shiv07tiwari** [comment]
screenshot_20180720_183248
screenshot_20180720_183251

**Comment by Swati4star** [comment]
The UI doesn't look good at all. First of all, change the icons size. They are so big.

And the UI is so much orange in color. You don't have to make it this much orange.

**Comment by shiv07tiwari** [comment]
What changes should I make in the UI ? I am really bad at designing work..so i need some suggestions

File: app/src/main/res/layout/fragment_about_us.xml [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+                      android:layout_height="match_parent"
+                      android:text="@string/version_text"/>
+                  <TextView
+                      android:layout_width="wrap_content"
+                      android:textSize="15sp"
+                      android:textColor="@color/mb_white"
+                      android:layout_marginStart="20dp"
+                      android:gravity="center"
+                      android:layout_height="match_parent"
+                      android:text="@string/version_value"/>
```
Comment (Swati4star) [inline app/src/main/res/layout/fragment_about_us.xml]: Don't add this as a contant. See this https://stackoverflow.com/a/4929559/3623128

File: app/src/main/java/swati4star/createpdf/fragment/AboutUsFragment.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View rootview = inflater.inflate(R.layout.fragment_about_us, container, false);
+        ButterKnife.bind(this, rootview);
+        return rootview;
+    }
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/AboutUsFragment.java]: Leave a line after each function

**Comment by shiv07tiwari** [comment]
Ya did that..now it is showing version name.
I need some UI suggestions

**Comment by Swati4star** [comment]
You can create an UI like this:

42753175-ad91f072-890d-11e8-9a69-212be87eab61

in the app description, write `An easy to use Android application to convert your files to PDF.`

**Comment by shiv07tiwari** [comment]
Made the changes @Swati4star 
screenshot_20180720_202522

**Comment by Swati4star** [comment]
- Remove `Report a bug`
- Remove `Feedback` from navigation drawer now
- Center align the app description
- Instead of `View on play store`, make it `Rate us on play store` 
- Remove `Rate us` from navigation drawer

**Comment by shiv07tiwari** [comment]
Made the changes @Swati4star

**Pull request merged**